### ISSUE TITLE
"RegexNotFoundError: Unable to extract title" for some bandcamp album page

### ISSUE THREAD

**Issue by Zerwan** [rg3/youtube-dl#4721]
There seems to be an issue when trying to extract the title of the songs of this bandcamp page: http://benprunty.bandcamp.com/album/ftl-advanced-edition-soundtrack

I obtain this error: "RegexNotFoundError: Unable to extract title"

Here is the complete output:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[xxx@yyy tmp]$ youtube-dl --verbose -o "%(autonumber)s - %(title)s.%(ext)s" --autonumber-size 2 --audio-format vorbis http://benprunty.bandcamp.com/album/ftl-advanced-edition-soundtrack
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', '-o', '%(autonumber)s - %(title)s.%(ext)s', '--autonumber-size', '2', '--audio-format', 'vorbis', 'http://benprunty.bandcamp.com/album/ftl-advanced-edition-soundtrack']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.01.11
[debug] Python version 3.4.2 - Linux-3.18.2-2-ARCH-x86_64-with-arch
[debug] exe versions: ffmpeg 2.5.3, ffprobe 2.5.3, rtmpdump 2.4
[debug] Proxy map: {}
[Bandcamp:album] ftl-advanced-edition-soundtrack: Downloading webpage
ERROR: Unable to extract title; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/youtube_dl/YoutubeDL.py", line 600, in extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python3.4/site-packages/youtube_dl/extractor/common.py", line 260, in extract
    return self._real_extract(url)
  File "/usr/lib/python3.4/site-packages/youtube_dl/extractor/bandcamp.py", line 164, in _real_extract
    title = self._search_regex(r'album_title : "(.*?)"', webpage, 'title')
  File "/usr/lib/python3.4/site-packages/youtube_dl/extractor/common.py", line 516, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract title; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

And thanks again for youtube-dl, it's awesome :)

**Comment by phihag** [comment]
Thank you for the report. I believe this issue to be fixed in youtube-dl 2015.01.16. See [our FAQ](https://yt-dl.org/update) if you need help updating.

**Comment by Zerwan** [comment]
Great, I'll test that as soon as the update reaches me. Thanks a lot!
